src/Tizen.NUI/src/public/BaseComponents/TextFieldEvent.cs

```diff
@@ -268,6 +322,27 @@ public TextField TextField
             }
         }
 
+        /// <summary>
+        /// The CursorMoved event arguments.
+        /// </summary>
+        public class CursorMovedEventArgs : EventArgs
```

Comment:
Do not use nested class

❌ DO NOT use public nested types as a logical grouping construct; use namespaces for this.
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/nested-types